java/spacewalk-java.spec

```
@@ -488,10 +488,7 @@ if [[ ! `java --list-modules | grep com.sun.xml.bind` ]]; then
 fi
 
 %install
-PRODUCT_NAME="SUSE Manager"
-%if !0%{?sle_version} || 0%{?is_opensuse} || 0%{?rhel} || 0%{?fedora}
-PRODUCT_NAME="Uyuni"
-%endif
+PRODUCT_NAME=%{productprettyname}
```

You miss a quote here too

```suggestion
PRODUCT_NAME="%{productprettyname}"
```